src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -172,38 +179,48 @@ def get_profile(
             NamedTuple (data, name, secure_props_not_found)
         """
 
+        logger = logging.getLogger(__name__)
+
         cfg_profile = Profile()
         try:
             cfg_profile = cfg.get_profile(
                 profile_name=profile_name, profile_type=profile_type, validate_schema=validate_schema
             )
         except jsonschema.exceptions.ValidationError as exc:
+            logger.error(f"Instance was invalid under the provided $schema property, {exc}")
             raise jsonschema.exceptions.ValidationError(
                 f"Instance was invalid under the provided $schema property, {exc}"
             )
         except jsonschema.exceptions.SchemaError as exc:
+            logger.error(f"The provided schema is invalid, {exc}")
             raise jsonschema.exceptions.SchemaError(f"The provided schema is invalid, {exc}")
         except jsonschema.exceptions.UndefinedTypeCheck as exc:
+            logger.error(f"A type checker was asked to check a type it did not have registered, {exc}")
             raise jsonschema.exceptions.UndefinedTypeCheck(
                 f"A type checker was asked to check a type it did not have registered, {exc}"
             )
         except jsonschema.exceptions.UnknownType as exc:
+            logger.error(f"Unknown type is found in schema_json, exc")
```

Comment:
I see that the string here is defined as an f-string - I think there are missing curly braces here:
```suggestion
            logger.error(f"Unknown type is found in schema_json, {exc}")
```